lib/utils/speech-utils.js

```diff
@@ -203,6 +203,29 @@ const testWellSaidTts = async(logger, credentials) => {
   }
 };
 
+const testElevenlabs = async(logger, credentials) => {
+  const {api_key, model_id} = credentials;
+  try {
+    const post = bent('https://api.elevenlabs.io', 'POST', 'buffer', {
+      'xi-api-key': api_key,
+      'Accept': 'audio/mpeg',
+      'Content-Type': 'application/json'
+    });
+    const mp3 = await post('/v1/text-to-speech/21m00Tcm4TlvDq8ikWAM', {
```

Comment:
I don't think we should hardcode voice id 21m00Tcm4TlvDq8ikWAM, instead use GET /Voices and just take the first one.  They remove a voice id in the future